### DISCUSSION TITLE
Public Metrics

### DISCUSSION THREAD

**Discussion by anupamvs** [tweepy/tweepy#1568] [Questions] [answered]
Is there any method in API to get the tweet's public metrics?

**Comment by Harmon758** [comment, marked as answer]
There is no `API` method or corresponding standard v1.1 API endpoint to do so, no.
However, it's possible to do so with the v2 API.
See https://github.com/tweepy/tweepy/issues/1472 and https://developer.twitter.com/en/docs/twitter-api/metrics.
